frontend/src/pages/learn/from-work-to-retirement.tsx

```diff
@@ -0,0 +1,175 @@
+import { FC, PropsWithChildren, useMemo } from 'react'
+
+import { Link as MuiLink } from '@mui/material'
+import { GetServerSideProps } from 'next'
+import { Trans, useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { NextSeo } from 'next-seo'
+
+import {
+  LearnMoreLink,
+  LearnPageLayout,
+} from '../../components/LearnPageLayout'
+
+export interface ImportantCardProps extends PropsWithChildren { }
+const ImportantCard: FC<ImportantCardProps> = ({ children }) => (
+  <p className="rounded-lg bg-[#ffe8a3]/[.3] p-5">{children}</p>
+)
+
+export interface DisclaimerCardProps extends PropsWithChildren { }
+const DisclaimerCard: FC<ImportantCardProps> = ({ children }) => (
+  <p className="rounded-lg bg-[#e9f1ff]/[.7] p-5">{children}</p>
+)
+
+
+const FromWorkToRetirement: FC = () => {
+  const { t } = useTranslation('learn/from-work-to-retirement')
+
+  const learnMoreLinks = useMemo<ReadonlyArray<LearnMoreLink>>(
```

Comment:
Don't need this variable, pass an empty array `[]` to `learnMoreLinks` prop.

Reply:
Thanks! Will make the changes.